test/dynamo/test_dynamo2.py

```diff
@@ -0,0 +1,31 @@
+import unittest
```

Comment:
This file needs to be linked into at least CPU CI by putting it in `test/run_test.sh` somewhere

Reply:
NA anymore